apisix/plugins/tcp-logger.lua

```diff
@@ -106,7 +108,7 @@ local function remove_stale_objects(premature)
 
     for key, batch in ipairs(buffers) do
         if #batch.entry_buffer.entries == 0 and #batch.batch_to_process == 0 then
-            core.log.debug("removing batch processor stale object, route id:", tostring(key))
+            core.log.warn("removing batch processor stale object, route id:", tostring(key))
```

Comment:
The "key" is no longer route id?

Reply:
@spacewander Nice capture, already fixed.